winbuild/build_prepare.py

```diff
@@ -456,7 +456,7 @@ def build_pillow():
         cmd_cd("{pillow_dir}"),
         *prefs["header"],
         cmd_set("DISTUTILS_USE_SDK", "1"),  # use same compiler to build Pillow
-        cmd_set("MSSdk", "1"),  # for Python 3.5 and PyPy3.6
+        cmd_set("MSSdk", "1"),  # for PyPy3.6
         cmd_set("py_vcruntime_redist", "true"),  # use /MD, not /MT
```

Comment:
Just to note here so I don't forget:

`MSSdk` and `py_vcruntime_redist` will [no longer be required by PyPy](https://foss.heptapod.net/pypy/pypy/-/commit/1f5925b68f4210fbc5d852b03643c588d45f99a5) in the next release.

`py_vcruntime_redist` is still [required by CPython 3.6](https://github.com/python/cpython/blob/46cbf6148a46883110883488d3e9febbe46ba861/Lib/distutils/_msvccompiler.py#L268), but [not CPython 3.7](https://github.com/python/cpython/blob/c8c818b0d73680516d5841597b705a1feeb42113/Lib/distutils/_msvccompiler.py#L242).